src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -403,16 +427,24 @@ public boolean isMergeable() {
     }
 
     public String getTarget() {
```

Comment:
the call to getPullRequest is using the PR object that we already have as the storage mechanism, instead of copying a reference to it here.  Just trying to minimize the storage space required as part of the trigger descriptor.  The call to getBase and getRef don't make live calls to github.